static/css/mozsearch.css

```diff
@@ -1819,3 +1826,21 @@ g.true-diag g.node polygon {
 #query-debug-results-json[aria-hidden] {
   display: none;
 }
+
+.search-warning {
+  margin: 8px 0;
+  padding-left: 10px;
+  border-left: 4px solid var(--search-warning-border-color);
+  font-size: 1rem;
+}
+
+.search-warning strong {
+  display: block;
+  margin-bottom: 2px;
+  font-weight: 600;
+}
+
+.search-warning-note {
+  font-size: 0.875rem;
```

Comment:
This seems to be too small.
(see the attached screenshot)
maybe we don't need resizing it?